src/components/CodeBlock.vue

```diff
@@ -52,8 +54,14 @@ export default {
     const getLineNumbers = computed(() => {
       return code.value.split('\n').length
     })
+    const copyCode = () => {
+      navigator.clipboard.writeText(code.value)
+      copyText.value = 'Copied'
+      setTimeout(() => (copyText.value = 'Copy'), 3000)
+      // alert('Code has been copied to clipboard.')
```

Comment:
@vfdev-5 How do we want to show Code is being copied?
Currently, it changes "Copy" to "Copied" once clicked and revert back to "Copy" after 3 seconds. OR do you want to give an alert that code has been copied?

Reply:
Tried "alert mode" locally and it's annoying when we click "Copy" again and again.